router/transformer/transformer.go

```diff
@@ -306,6 +284,6 @@ func getRouterTransformURL() string {
 	return strings.TrimSuffix(config.GetEnv("DEST_TRANSFORM_URL", "http://localhost:9090"), "/") + "/routerTransform"
 }
 
-func getResponseTransformURL(destName string) string {
-	return strings.TrimSuffix(config.GetEnv("DEST_TRANSFORM_URL", "http://localhost:9090"), "/") + "/transform/" + strings.ToLower(destName) + "/response"
+func getProxyURL(destName string) string {
+	return strings.TrimSuffix(config.GetEnv("DEST_TRANSFORM_URL", "http://localhost:9090"), "/") + "/v0/destinations/" + strings.ToLower(destName) + "/proxy"
```

Comment:
Are we using the same DEST_TRANSFORM_URL since I assumed that we would be having different pods which handle the requests for transformer proxy

Reply:
No, currently we went with transformer as a single service for proxy, as per discussion cc @sanpj2292